### PR TITLE
Build revamp

### PR DESCRIPTION
**Pull request by dkarlovi**
Introducing PHPStan and updating CS Fixer.

Note; this PR has a large number of files changed because of CS fixer changes (adding strict types, for one).

### CODE REVIEW

File: Form/Handler/AuthorizeFormHandler.php [hunk 1]
```diff
@@ -51,6 +53,15 @@ public function __construct(FormInterface $form, $requestStack = null)
         $this->requestStack = $requestStack;
     }
 
+    public function __get($name)
+    {
+        if ($name === 'request') {
+            @trigger_error(sprintf('%s::$request is deprecated since 1.4 and will be removed in 2.0.', __CLASS__), E_USER_DEPRECATED);
```
Comment (dkarlovi) [inline Form/Handler/AuthorizeFormHandler.php]: Actually remove.

**Comment by dkarlovi** [comment]
@wackyMole you could take a look at this and suggest improvements, it looks good enough to me.

**Comment by dkarlovi** [comment]
@wackyMole we are working on a major release, points you made are all valid and should get their own PR.

My question was more about this PR. If you don't have any objections, that answers my question. :+1: 

@dinamic how about you? This is supposed to be merged into master where you're working on those integration tests.

**Comment by dinamic** [comment]
Still trying to minimize the refactoring impact. Some areas are so hardly testable, I wanna rewrite them. AuthorizeController is a good example.

**Comment by dkarlovi** [comment]
OK, if I'm reading your responses correctly, you both agree (or at least don't disagree) to merge this. :+1:

**Comment by dinamic** [comment]
You have a green light from me. I will rebase my changes afterwards.

**Comment by wackyMole** [comment]
:+1:

**Pull request merged**